src/Microsoft.PowerShell.Commands.Management/commands/management/Process.cs

```diff
@@ -4,6 +4,7 @@
 using System;
```

Comment:
@SteveL-MSFT, your last commit had 3 failures in `PowerShell-CI-windows`
WindowStyle argument.-WindowStyle Minimized should work on Windows
```powershell
Expected exactly 'Minimized', but got Hidden.
at <ScriptBlock>, D:\a\1\s\test\powershell\Host\ConsoleHost.Tests.ps1: line 890
890:             $showCmd | Should -BeExactly $WindowStyle
```
WindowStyle argument.-WindowStyle Maximized should work on Windows
```powershell
Expected exactly 'Maximized', but got Hidden.
at <ScriptBlock>, D:\a\1\s\test\powershell\Host\ConsoleHost.Tests.ps1: line 890
890:             $showCmd | Should -BeExactly $WindowStyle
```
XmlCommand DRT basic functionality Tests.Import-Clixml should work with XML serialization from pwsh.exe
```powershell
Data at the root level is invalid. Line 1, position 1.
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Utility\XMLCommand.Tests.ps1: line 110
110: 		$out = Import-Clixml -Path $testfile

```